### ISSUE TITLE
Did not Receive key item after killing Shadow Lord

### ISSUE THREAD

**Issue by agjevik999** [EdenServer/community#1645]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x ] I have provided my client version in the details. (type /ver into your game window)

### Details

Client Version:  30181025_0

I can't reproduce this bug, because I cant redo the Battle Fight atm.

I was in a group this evening and we all defeated the Shadow Lord. I was doing Sandy Mission 5-2, when I was portaled out of the castle after all the cutscenes I did not obtain the Key Item required to complete the mission. As such when I went back to talk to Halver, I would not get any dialogue and I am stuck. 

I have attached screenshots showing that I was in the group when the Shadow Lord was killed, as well as the dialogue for the cut scene that follows winning the fight. Finally I attached a screenshot of myself with the title you receive when beating the fight.

![Osopi_2019 05 13_225625](https://user-images.githubusercontent.com/49248117/57668396-2d4cc300-75d5-11e9-9f28-c678e800913c.png)
![Osopi_2019 05 13_225629](https://user-images.githubusercontent.com/49248117/57668397-2de55980-75d5-11e9-9cc4-b376ac21ea45.png)
![Osopi_2019 05 13_225632](https://user-images.githubusercontent.com/49248117/57668398-2de55980-75d5-11e9-86be-736f9559c65f.png)
![Osopi_2019 05 13_225913](https://user-images.githubusercontent.com/49248117/57668399-2de55980-75d5-11e9-8997-c0fcefbbb2ad.png)

**Comment by agjevik999** [comment]
Moire Screens to show that I was taking damage and in the party when both forms of the boss was killed.


![Osopi_2019 05 13_234106](https://user-images.githubusercontent.com/49248117/57669376-b87b8800-75d8-11e9-83d2-de83cf7aa5ce.png)
![Osopi_2019 05 13_234109](https://user-images.githubusercontent.com/49248117/57669377-b87b8800-75d8-11e9-9fbc-b69c0d067ee8.png)
![Osopi_2019 05 13_234111](https://user-images.githubusercontent.com/49248117/57669378-b87b8800-75d8-11e9-83d2-56f689f9f366.png)
![Osopi_2019 05 13_234113](https://user-images.githubusercontent.com/49248117/57669379-b87b8800-75d8-11e9-826c-2f5458de77c0.png)

**Comment by godmode** [comment]
You likely needed to examine the door to get the cutscene before the battlefield status started.

**Comment by waffleness** [comment]
I was in Osopi's group for this. Here was the order of events:

6 of us were in a group and Osopi was in an alliance with us by himself.

All 7 of us viewed the first cutscene when examining the door.
Between phase 1 and phase 2, all 7 of us viewed the mid fight cutscene, even Osopi who was not currently in battle. 

After victory, the 6 in battle received the cutscene, key item, and were warped to the bottom of the castle. Osopi did not receive any of the above. 

5 of us climbed back up and joined a party with Osopi.

Examining the door, Osopi did not see the cutscene (he already had) and we entered the battlefield.

Upon completion, Osopi received the cutscene and was warped to the bottom of the castle, but did not receive the key item.

Perhaps there is an issue when in an alliance like we did it? As far as I could tell, Osopi did everything correctly and saw all cutscenes as intended, but just did not receive the KI.

**Comment by agjevik999** [comment]
Yes, i couldnt really have said it better. That description was exactly the way it happened.

**Comment by agjevik999** [comment]
Id like to just add to my bug report, as I seem to have found a related bug. I still havent neen able to get the key item and be able to complete the mission, so I remain Rank 5 for the moment. 

I zoned into Norg today for reasons unrelated to this bug report, and I got the cutscene to start the ZM missions. I linked the mission from the wikipedia, and according to the "Upon Completion" section of the wiki page, I should only have gotten this cutscene after I got my rank 6.


https://ffxiclopedia.fandom.com/wiki/The_Shadow_Lord
